### DISCUSSION TITLE
Why does `VList` have no `as` prop?

### DISCUSSION THREAD

**Discussion by chloe463** [inokawa/virtua#502] [Q&A] [answered]
Hi!
Thanks for the great library. It helps me a lot.
However, I have a question about `VList`.
Is there any reason that `VList` doesn't have `as` prop, whereas `WindowVirtualizer` have it?
In my use case, I want to use `VList` with `ul` as root and `li` as item, but `VList` doesn't have `as` props so I can't achieve that goal.

```
<VList
  count={count}
  overscan={10}
  as="ul" // <- It is not possible 🤔
  item="li"
>...</VList>
```

**Comment by inokawa** [comment, marked as answer]
You can use `Virtualizer` instead of `VList`. `VList` is just a wrapper of `Virtualizer` so you can build your own VList by yourself.
https://github.com/inokawa/virtua/blob/63e9285cb723055aa278c88122c73d3e67015f90/src/react/VList.tsx#L108-L121

Customizing the inner div seems to be an advanced usecase, probably ul, dl and table only need it. We think `VList` is for virtualization with minimal settings, to cover 80~90% of usecases, so we'd like to keep its API less.

**Reply by chloe463** [reply]
@inokawa 
Thank you for the reply!
I think I can make it with `Virtualizer`. And I understand your thought about the `VList` 👍 
Thank you again.
